video/out/vo_gpu_next.c

```diff
@@ -994,14 +994,20 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
     if (!should_draw || !pl_swapchain_start_frame(p->sw, &swframe)) {
         if (frame->current) {
             // Advance the queue state to the current PTS to discard unused frames
+#if PL_API_VER >= 340
             pl_queue_update(p->queue, NULL, pl_queue_params(
                 .pts = frame->current->pts + pts_offset,
                 .radius = pl_frame_mix_radius(&params),
                 .vsync_duration = can_interpolate ? frame->ideal_frame_vsync_duration : 0,
-#if PL_API_VER >= 340
                 .drift_compensation = 0,
-#endif
             ));
+#else
+            pl_queue_update(p->queue, NULL, pl_queue_params(
+                .pts = frame->current->pts + pts_offset,
+                .radius = pl_frame_mix_radius(&params),
+                .vsync_duration = can_interpolate ? frame->ideal_frame_vsync_duration : 0,
+            ));
```

Comment:
ditto

Reply:
I think we should keep using macro. This is haasn way of injecting default values. While I don't like it that much, we should keep it.

What I meant is to move `drift_compensation` outside, instead of duplicating struct declaration

Example:
``` diff
diff --git a/video/out/vo_gpu_next.c b/video/out/vo_gpu_next.c
index 0e42e19085..af28778f96 100644
--- a/video/out/vo_gpu_next.c
+++ b/video/out/vo_gpu_next.c
@@ -1029,10 +1029,10 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
             .radius = pl_frame_mix_radius(&params),
             .vsync_duration = can_interpolate ? frame->ideal_frame_vsync_duration : 0,
             .interpolation_threshold = opts->interpolation_threshold,
+        );
 #if PL_API_VER >= 340
-            .drift_compensation = 0,
+        qparams.drift_compensation = 0,
 #endif
-        );

         // Depending on the vsync ratio, we may be up to half of the vsync
         // duration before the current frame time. This works fine because
```